distribution/docs/src/main/resources/_contents/_configuring/configuring-global-settings-contents.adoc

```diff
@@ -103,30 +103,48 @@ Does not change the address the system runs on.
 |HTTPS Port
 |org.codice.ddf.system.httpsPort
 |String
-|The https port used by the system.
+|The https port used to advertise the system.
 
-NOTE: This *DOES* change the port the system runs on.
+NOTE: This does not change the port the system runs on.
 |8993
 |Yes
 
 |HTTP Port
 |org.codice.ddf.system.httpPort
 |String
-|The http port used by the system.
+|The http port used to advertise the system.
 
-NOTE: This *DOES* change the port the system runs on.
+NOTE: This does not change the port the system runs on.
 |8181
 |Yes
 
 |Default Port
 |org.codice.ddf.system.port
 |String
-|The default port used to advertise the system. This should match either the http or https port.
+|The default port used to advertise the system. This should match either the above http or https port.
 
 NOTE: Does not change the port the system runs on.
 |8993
 |Yes
 
+|Internal HTTPS Port
```

Comment:
❓ Should we add a note for those new properties saying that the internal ports should be the same as the external ones unless the system is behind a proxy or load-balancer? Just thinking that we need to make it clear that in a simple, stand-alone configuration these values need to default to the external ports for things to work.